test/integration/tests/utilities.js

```diff
@@ -59,6 +59,7 @@
       return networkStore.save(entity);
     }))
       .then(() => syncStore.pull())
+      .then(() => syncStore.find().toPromise())
```

Comment:
@tsvetomir-nedyalkov please have a look at this and make sure it won't change the behaviour of the method. The result type of pull() has changed and it shouldn't be used like it is being used here. Yes, it is a breaking change, but it's been discussed and approved.